src/silly_sprites/opengl/window.cpp

```diff
@@ -43,8 +43,8 @@ namespace sly::gl {
             return tl::unexpected{ GlError::FailedToInitializeGlad };
         }
 
-        auto const callback{ [](GLFWwindow* const window, int const width, int const height) {
```

Comment:
do we name variables in parameter lists event when we don't use them? 
in cpp and lambas we could leave the names out.
or mark them as `[[maybe_unused]]`
just du indecate that they are not used and to reduce warnings.